pulser-core/pulser/backend/qpu.py

```diff
@@ -40,11 +49,26 @@ def run(
                 available.  If set to False, the call is non-blocking and the
                 obtained results' status can be checked using their `status`
                 property.
+            open_submission: A flag indicating whether or not the submission
+                should be for a single job or accept future jobs before
+                closing.
+            submission_id: If you have a preexisting submission and wish to add
+                more jobs to it then you can provide the ID here and 'run'
+                will attempt allocate jobs.
```

Comment:
I feel this option is misplaced here... To me, it would make more sense to have a different method for adding jobs to an open submission. 

Another thought: would it make sense if a backend kept track of an open submission, disallowing you to make a new submission before the former is closed? What's the harm if a user forgets to close an open submission?

Reply:
Yeah I think so too now. My initial thought was `run()` is a generic term for executing something, so reusing it wouldn't be outrageous but, I can move the features into another method